README.md

```diff
@@ -1,19 +1,22 @@
-# qMRLab (beta)
-[![Build Status](https://travis-ci.org/qMRLab/qMRLab.svg?branch=master)](https://travis-ci.org/qMRLab/qMRLab) [![Coverage Status](https://coveralls.io/repos/github/qMRLab/qMRLab/badge.svg?branch=master)](https://coveralls.io/github/qMRLab/qMRLab?branch=master) [![Documentation Status](https://readthedocs.org/projects/pip/badge/?version=latest)](https://qmrlab.readthedocs.io/en/master/?version=latest) [![License](https://img.shields.io/github/license/mashape/apistatus.svg)](https://opensource.org/licenses/MIT) [![Website](https://img.shields.io/badge/Website-qmrlab.org-red.svg)](https://qmrlab.org) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/neuropoly/qMRLab/bids?filepath=qMRLab/mt_sat_example.ipynb) [![DockerHub](https://img.shields.io/badge/DockerHub-qmrlab-blue.svg)](https://hub.docker.com/r/neuropoly/qmrlab/) [![Twitter Follow](https://img.shields.io/twitter/follow/espadrine.svg?style=social&label=Follow)](https://twitter.com/qmrlab)
+# [qMRLab (beta)](https://qmrlab.org)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [![Build Status](https://travis-ci.org/qMRLab/qMRLab.svg?branch=master)](https://travis-ci.org/qMRLab/qMRLab) [![Coverage Status](https://coveralls.io/repos/github/qMRLab/qMRLab/badge.svg?branch=master)](https://coveralls.io/github/qMRLab/qMRLab?branch=master) [![Documentation Status](https://readthedocs.org/projects/pip/badge/?version=latest)](https://qmrlab.readthedocs.io/en/master/?version=latest) [![License](https://img.shields.io/github/license/mashape/apistatus.svg)](https://opensource.org/licenses/MIT) [![Website](https://img.shields.io/badge/Website-qmrlab.org-red.svg)](https://qmrlab.org) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/neuropoly/qMRLab/bids?filepath=qMRLab/mt_sat_example.ipynb) [![DockerHub](https://img.shields.io/badge/DockerHub-qmrlab-blue.svg)](https://hub.docker.com/r/neuropoly/qmrlab/) [![Twitter Follow](https://img.shields.io/twitter/follow/espadrine.svg?style=social&label=Follow)](https://twitter.com/qmrlab)
 
-<a href="https://qmrlab.org" target="_blank"><img src="https://github.com/qMRLab/qMRLab/blob/master/docs/logo/page_header.png?raw=true" width = "1000" height="650" align="middle"></a>
+<a href="https://qmrlab.org" target="_blank"><img src="/docs/logo/page_header.png" width = "1100" height="650" align="middle"></a>
 
-<p align="center"><h2> qMRLab is an open-source software for quantitative MR image analysis.</h2></p>
+<h2>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; qMRLab is an open-source software for quantitative MR image analysis.</h2>
```

Comment:
I'm afraid that this hack to "center" this line might not work well on mobile.

Reply:
Resolved.

Reply:
Markdown image embed automatically preserves aspect ratio, looks good on mobile now.